runtime_lib/CMakeLists.txt

```diff
@@ -39,7 +38,19 @@ foreach(target ${AIE_RUNTIME_TARGETS})
   message("Building AIE runtime for ${target} with TOOLCHAIN=${${target}_TOOLCHAIN_FILE}")
   
   # xaiengine
-  ExternalProject_Add(xaiengine_${target}
+  # if no LibXAIE_{target}_DIR specified, build xaiengine_${target} from Vitis
+  if (DEFINED LibXAIE_${target}_DIR)
+    message("Using xaiengine from LibXAIE_${target}_DIR=${LibXAIE_${target}_DIR}.")
+    #set(xaiengine_${target} ${LibXAIE_${target}_DIR})
+    ExternalProject_Add(xaiengine_${target}
```

Comment:
Why do you need an externa project, if this is already built somewhere?  I'd think if you have something already built, you'd need some install rules?

Reply:
My preference would be to point to an existing install somewhere, no building or copying needed, like we had in the past.